tasks/sass.js

```diff
@@ -45,8 +44,7 @@ gulp.task('components:sass', function() {
 
     return gulp
       .src([
-        config.global.src + config.global.components + '/**/*.scss',
-        '!' + config.global.src + config.global.components + '/**/_*.scss'
```

Comment:
why are you removing this line? not sure what this makes regarding backward compatability

Reply:
i think backwards it should be fine, it will output more files than before i think. That needs to be done for "cacheability" i guess?

Reply:
There should be no issues with backwards compatability. 
By caching all the files the sass files importing the partials won't update when a partial is changed.
For this i added the gulp-dependents plugin which checks the files for imports and triggers an update if a partial is changed. In order for this to work the partials need to be in the gulp stream. 
Since gulp doesn't compile files starting with a _, this shouldn't slow down compilation time.
For the serve process the compilation time using caching and depenents is reduced significantly. 
I will check if it influences the build process since that one will not use caching.

Reply:
I just checked between 5.4.2 and this version and there is no difference in the build process timing.